dev/lint-python

```diff
@@ -275,8 +275,7 @@ SPARK_ROOT_DIR="$(dirname "${SCRIPT_DIR}")"
 
 pushd "$SPARK_ROOT_DIR" &> /dev/null
 
-# skipping local ruby bundle directory from the search
-PYTHON_SOURCE="$(find . -path ./docs/.local_ruby_bundle -prune -false -o -name "*.py")"
+PYTHON_SOURCE="$(git ls-files '*.py')"
```

Comment:
This is an improvement, but I don't think this is that great, either. In general, I don't like that we are building a huge string of every Python file and passing it as an argument.

Instead, we should using the appropriate include and exclude filters (e.g. via tox.ini) to capture everything to be tested. But I don't want to get into that here.